hack/ci/check-upgrade.sh

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env bash
+
+# This file is part of MinIO Direct CSI
+# Copyright (c) 2021 MinIO, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#
+# This script is indented to run in Github workflow.
+# DO NOT USE in other systems.
+#
+
+if [[ $# -ne 1 ]]; then
+    echo "error: image tag must be provided"
+    echo "usage: $(basename "$0") <IMAGE_TAG>"
+    exit 255
+fi
+
+set -ex
+
+IMAGE_TAG="$1"
+LV_DEVICE=""
+LUKS_DEVICE=""
+DRIVES_COUNT=0
+VOLUMES_COUNT=0
+
+function create_loop() {
+    truncate --size="$2" "$1"
+    sudo losetup --find "$1"
+    sudo losetup --noheadings --output NAME --associated "$1"
+}
+
+function setup_lvm() {
+    loopdev=$(create_loop testpv.img 1G)
+    sudo pvcreate "$loopdev"
+    vgname="testvg$RANDOM"
+    sudo vgcreate "$vgname" "$loopdev"
+    sudo lvcreate --name=testlv --extents=100%FREE "$vgname"
+    LV_DEVICE=$(readlink -f "/dev/$vgname/testlv")
+}
+
+function setup_luks() {
+    loopdev=$(create_loop testluks.img 1G)
+    echo "mylukspassword" > lukspassfile
+    yes YES | sudo cryptsetup -y -v luksFormat "$loopdev" lukspassfile
+    sudo cryptsetup -v luksOpen "$loopdev" myluks --key-file=lukspassfile
+    LUKS_DEVICE=$(readlink -f /dev/mapper/myluks)
+}
+
+function install_directcsi_v1.3.6() {
```

Comment:
As installation is generic for current and previous releases, have a version independent function for install/uninstall.